spdmlib/src/responder/context.rs

```diff
@@ -62,6 +56,7 @@ impl<'a> ResponderContext<'a> {
     pub fn process_message(
         &mut self,
         timeout: usize,
+        auxiliary_app_data: &[u8],
```

Comment:
Since you need to add additional parameters `auxiliary_app_data`, why not use callback directly?

For example: process_message(self, timeout, dispatch_secured_app_message_cb, auxiliary_app_data)